iset.mm

```diff
@@ -35021,6 +35021,7 @@ conditioning it (with ` x e. z ` ) so that it asserts the existence of a
      depend on the Axiom of Regularity.  (Contributed by NM, 23-Aug-1993.) $)
   vprc $p |- -. _V e. _V $=
     ( vx cvv wcel cv wceq wex vnex isset mtbir ) BBCADBEAFAGABHI $.
+  $( $j usage 'vprc' avoids 'ax-setind'; $)
```

Comment:
ax-setind (#4381) is located behind vprc (#3992) so this is somewhat questionable.  Why adding the $j in such a case?

Reply:
I kind of lost track about whether we made any decision about this.

The reason to add it is that it expresses an intention that might help in the future as people re-order theorems.

The reason not to add it is that it is, as you point out, redundant with the position in the file.

I guess I'll remove it at least for now. I don't have an especially strong opinion.